pybeautifier.py

```diff
@@ -152,6 +154,26 @@ def serve():
         server.close()
 
 
+def read_message(client) -> str:
+    data = ''
+    while True:
+        try:
+            client.setblocking(0)
+            (r, w, x) = select([client], [], [], 1)
+            if client not in r:  # chunk == ''
+                break
+            chunk = client.recv(4096)  # some 2^n number
+            if not chunk:
+                break
+            data += chunk.decode('utf-8')
```

Comment:
Should decode after received all data bytes, otherwise decode non-complete bytes may failed.

Reply:
Good catch, thanks.